_themes/conan/layout.html

```diff
@@ -162,6 +162,10 @@
               Come and join <b><a href=https://conandays.conan.io>ConanDays</a></b>, a technical event bringing together all worldwide Conan users to share their experiences and contribute to the future of Conan. <b><a href=https://blog.conan.io/2019/11/05/ConanDays-Madrid-2020-international-community-gathering-Conan-package-manager.html>Read more about it here</a></b>. It will take place in Madrid, Spain, on March 19-20, 2020. <b><a href=https://sessionize.com/conandays-2020/>Submit a talk</a></b> and contribute to the community sharing your knowledge or <b><a href=https://www.eventbrite.com/e/conandays-2020-gathering-the-clans-tickets-75869891889>reserve your spot now!</a></b>
           </div>
 
+          <div class="admonition warning">
```

Comment:
Hi @SSE4 

This is too much, as a general announcement. Please change it in the "Installation" section only, I think it will be enough. I have also created a pinned issue in Conan repo.

Reply:
moved